resources/server_install.rb

```diff
@@ -24,7 +24,7 @@
 property :extconf_directory, String,        default: lazy { ext_conf_dir }
 property :data_directory,    String,        default: lazy { data_dir }
 property :external_pid_file, String,        default: lazy { "/var/run/mysql/#{version}-main.pid" }
-property :password,          [String, nil], default: 'generate'
+property :password,          [String, nil], default: lazy { node['mariadb']['server_root_password'] }
```

Comment:
We don't want to use attributes inside here, they are plain text on the server, please revert to a sane default ie `'generate'`

Reply:
ok I'll review the password generation code and see what is affected by that.

Reply:
everything should stay the same with that revert, however, I used `lazy { node['mariadb']['server_root_password'] }` as the default due to this misleading [comment](https://github.com/sous-chefs/mariadb/blob/master/README.md#syntax)

Reply:
I reverted that and removed from CHANGELOG.md indication about the `server_root_password` usage for the default value.